awslabs/performance-dashboard-on-aws
# Add frontend support for adding/updating a Google Analytics tracking ID

## Description

GTT-1938: Adds frontend support for analytics collection by enabling the addition/editing of a Google Analytics tracking ID.

## Testing
* Unit tests.
* New instance: https://d3iquritmke7bx.cloudfront.net/admin/settings/publishedsite
  - Click "Edit" in the "Web analytics" section.
  - Add/edit the tracking ID.
  - Valid tracking IDs begin with UA. Try saving an invalid ID.
  - Try clearing and saving.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
